### DISCUSSION TITLE
Exclude unwanted regions on 10x Visium images and compare annotated regions on images

### DISCUSSION THREAD

**Discussion by cf98** [satijalab/seurat#6739] [Q&A]
Hi everyone,

I am learning the 10X visium data analysis pipeline on Seurat.
I noticed that there are a few fold or wrinkles on my image, and I am wondering if Seurat would allow me to exclude those regions. It would be nice if there's an interactive interface to mark those regions and exclude them before starting on everything. 
Also, I hope to know if it's possible to overlay pathology annotations on the image. It would be very nice to see if an annotated region is enriched for specific gene expressions or cluster of cell types.

Thanks,
Changfeng

**Comment by WesDe** [comment]
Hi, 

You can use the subset function to remove or select region based on their location (information located in {slice_name}_imagecol and {slice_name}_imagerow).

Best,
